Lib/Protocols/IdFTP.pas

```diff
@@ -680,17 +680,44 @@   TIdFTPClientIdentifier = class (TPersistent)
     FClientName : String;
     FClientVersion : String;
     FPlatformDescription : String;
+    FVendor : String;
+    function GetCSIDOutput: String;
     procedure SetClientName(const AValue: String);
     procedure SetClientVersion(const AValue: String);
     procedure SetPlatformDescription(const AValue: String);
     function GetClntOutput: String;
   public
     procedure Assign(Source: TPersistent); override;
     property ClntOutput : String read GetClntOutput;
+    property CSIDOutput : String read GetCSIDOutput;
   published
     property ClientName : String read FClientName write SetClientName;
     property ClientVersion : String read FClientVersion write SetClientVersion;
-    property PlatformDescription : String read FPlatformDescription write SetPlatformDescription;
+    property Vendor : String read FVendor write FVendor;
+    property PlatformDescription : String read FPlatformDescription
+      write FPlatformDescription;

```

Comment:
Why was `Vendor` added before `PlatformDescription`? You removed `SetPlatformDescription()` from the `PlatformDescription` property setter, but did not remove `SetPlatformDescription()` itself from the unit.  This seems like a needless change. Should `Vendor` and `PlatformDescription` have setters to mirror the `ClientName` and ClientVersion` properties, which trim their input values?